.github/workflows/atlas-tests.yml

```diff
@@ -0,0 +1,61 @@
+## Github workflow to run atlas tests
+
+name: Tests::Atlas
+
+on:
+  workflow_call:
+
+defaults:
+  run:
+    shell: bash
+
+## env vars are transferred to composite action steps
+env:
+  BITCOIND_TEST: 1
+  RUST_BACKTRACE: full
+  SEGMENT_DOWNLOAD_TIMEOUT_MINS: 15
+  RETRIES: 3
+  RETRY_DELAY: 10000
+  TEST_TIMEOUT: 30
+  TEST_RETRIES: 2
```

Comment:
What's making use of the `RETRIES`, `RETRY_DELAY`, and `TEST_RETRIES` vars? I see [some of your composite actions](https://github.com/stacks-network/actions/blob/main/stacks-core/run-tests/action.yml) have the ability to accept these are parameters, but they don't seem to be getting passed along. Any insight here is appreciated!

Reply:
leftover from previous PR - i'll remove them. they're still *valid* but they should be inputs
ex: 
```yml
      ## Create and upload code coverage file
      - name: Code Coverage
        id: codecov
        uses: stacks-network/actions/codecov@main
        with:
          test-name: ${{ matrix.test-name }}
          retries: 5 # default is 3
          retry-delay: 50000 # default is 10000
  ```